packages/patternfly-4/src/content/design-guidelines/usage-and-behavior/charts/charts.md

```diff
@@ -200,3 +200,19 @@ The brush and zoom properties allow for magnification of an area, bar, line or s
 
 1. **Window:** Click and drag to create a window around the section of the chart that you want to zoom in on
 2. **Magnified chart:** The portion selected will appear above the full chart view.
+
+### Interactive chart legends
+For charts that have more than one data set graphed, interactive legends can be used to highlight or isolate certain data sets.  This would work well for [line](#line-chart), [area](#area-chart), or [stacked](#stacked-bar-chart) type of charts where overlap of data sets could occur.
```

Comment:
@nlcwong Add an "s" to the end of #stacked-bar-chart ! So it'll be [stacked](#stacked-bar-charts)

Reply:
I just updated the file to have the #stacked-bar-charts corrected.